src/db.ts

```diff
@@ -120,11 +120,7 @@ export const migrate = async () => {
       },
     };
 
-    await savePerson(person);
```

Comment:
A lot of mock-solaris code relies on this default person that uses an old schema, which means unexpected stuff may break if it doesn't exist. Can we not delete it?

Reply:
Sure!

Reply:
[38abdb5](https://github.com/etvascom/mock-solaris/pull/6/commits/38abdb54f76fe13caa3e931bedeee21e07e15fc5)

Reply:
Assuming we are using a real RedisDB, sure. But right now we are using an in-memory cache so we will seed on deploy always anyway. This code was searching for their specific person object before, and was somehow renamed to `demo100` during a refactor most probably. I don't think this will affect us.

Reply:
yeah .. but is still fundamentally wrong :)  .. why search for user100 if we only insert 10 ? could easily be fixed if we search for the first user or their default person 

also .. is there still a plan to create a MR to the fork origin repo ?

Reply:
Right, we should search for the default kontist user. 

There was a plan to create a MR, but I don't know if we are still aiming for it ... we would need to spend a few hours refactoring and I don't think it has high priority at the moment